multi: Expand the API to support retrieving paid amounts. (#1698)
decred/politeia: politeiawww/api/cms/v1/api.md

  }
```

### `Proposal Billed State`

> Retrieve proposal's publicly available payment information

Reviewer: ```suggestion
Retrieve proposal's publicly available payment information.
```

This retrieves the token of proposal that needs to get the payment data and 
uses that token to search through the database for invoices that have line-items 
containing the proposal token. Calculates and return only two pieces of 

> information: amount paid and payment time in the invoices

Reviewer: ```suggestion
This retrieves the proposal token that is needed to get the payment data and 
uses that token to search through the database for invoices that have line-items 
containing the proposal token. Calculates and returns only two pieces of 
information: amount paid and payment time.
```

Note: This call is public.

**Route:** `GET /v1/proposals/billedstate`

> | Parameter | Type | Description | Required |

Reviewer: ```suggestion
**Route:** `GET /v1/proposals/billedstate`

**Params:**

| Parameter | Type | Description | Required |
```
